app/assets/stylesheets/intl_tel_input.css

```diff
@@ -0,0 +1,4 @@
+/* Make international telephone input use the whole column */
```

Comment:
Each of the usecases of this file requires `intl-tel-input/build/css/intlTelInput` to be used as well, so it might be better if we make this file `*= require` that file here.

Reply:
`intl_tel_input.css` now requires `intl-tel-input/build/css/intlTelInput`